keylime/tenant.py

```
@@ -1114,8 +1109,12 @@ def do_cvdelete(self, verifier_check: bool = True) -> None:
                     self.verifier_fid_str,
                     numtries,
                 )
-                # Marked for deletion (need to modify the code on CI tests)
                 logger.info("Agent %s deleted from the CV", self.agent_uuid)
+        else:
+            raise UserError(
+                f"Unexpected response code {response_json['code']} from "
+                f"{self.verifier_fid_str} while deleting {self.agent_fid_str}"
+            )
```

Ruff (0.15.9)

[warning] Avoid specifying long messages outside the exception class

(TRY003)